src/Microsoft.AspNet.Razor.Runtime/Resources.resx

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+    
```

Comment:
As ugly as it seems I'd rather have the whitespace. Most people touching the resx files will be doing it via VS and then will end up with the same comment/same changeset (more work to do in the future).